Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -102,13 +102,13 @@ public Analytics(Config config, ImportedReferences references, ILogger logger, M
                     member is PropertyDefinition property &&
                     (property.PropertyType.IsIRealmObjectBaseImplementor(_references) ||
                     property.PropertyType.IsRealmObjectDescendant(_references)) ?
-                    (true, Feature.RealmObjectReference) : default,
-                [Feature.RealmValue] = member => (true, Feature.RealmValue),
+                    new ActionDriver(true, Feature.RealmObjectReference) : new ActionDriver(),
```

Comment:
These (and everywhere else) can be simplified by the new target-typed new expressions.